model/lib/email/email_action_type.dart

```diff
@@ -17,5 +17,6 @@ enum EmailActionType {
   selection,
   moveToSpam,
   unSpam,
-  openInNewTab
+  openInNewTab,
+  resend
```

Comment:
`resend`: this is not an action for `email`, it is only an action for `sending_email`